### PR TITLE
compatible with matplotlib 3.5.1

### PR DESCRIPTION
**Pull request by jiapei100**
compatible with matplotlib 3.5.1

### CODE REVIEW

**Comment by michalkurka** [comment]
@jiapei100 thank you for the contribution!

@koniecsveta @tomasfryda can you please review?

File: h2o-py/h2o/model/extensions/std_coef.py [hunk 1]
```diff
@@ -84,22 +84,22 @@ def _std_coef_plot(self, num_of_features=None, server=False, save_plot_path=None
             markers = [plt.Line2D([0, 0], [0, 0], color=color, marker="s", linestyle="")
                        for color in signage[0:num_of_features]]
             lgnd = plt.legend(markers, color_ids, numpoints=1, loc="best", frameon=False, fontsize=13)
-            lgnd.legendHandles[0]._legmarker.set_markersize(10)
+            lgnd.legendHandles[0].set_markersize(10)
```
Comment (sebhrusen) [inline h2o-py/h2o/model/extensions/std_coef.py:87]: for backwards compatibility, can you please instead try to set `markersize=10` in the `plt.Line2D` constructor above?
using constructor properties should work with most versions of `matplotlib` whereas using this setter works only from v3.5.0.

Comment (sebhrusen) [reply h2o-py/h2o/model/extensions/std_coef.py:87]: for example, if we want to set it only for the first marker:
```python
markers = [plt.Line2D([0, 0], [0, 0], 
                   color=color,
                   marker="s", 
                   markersize=10 if idx==0 else None,
                   linestyle="")
                   for idx, color in enumerate(signage[0:num_of_features])]
```

**Review by sebhrusen — CHANGES_REQUESTED** [review]
Thanks for raising the issue:
let's try to set `markersize` directly in the constructor.

**Pull request closed without merging**